File: src/main/cpp/kattis/oddities.cpp
#include<bits/stdc++.h>

using namespace std;

int main() {
    int tc;
    scanf("%d", &tc);
    
    while (tc--) {
        int n;
        scanf("%d", &n);
        if (n % 2)     printf("%d is odd\n", n);
        else           printf("%d is even\n", n);
    }
    return 0;
}

File: src/main/java/com/algorithms/halim/cap1/Oddities.java
package com.algorithms.halim.cap1;

import java.util.Scanner;

public class Oddities {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int TC = sc.nextInt();

        while (TC-- > 0) {
            int x = sc.nextInt();
            if (x % 2 == 0) System.out.printf("%d is even%n", x);
            else            System.out.printf("%d is odd%n", x);
        }
    }
}
